### PR TITLE
Add proper Home Manager module output (Updated #113)

### PR DESCRIPTION
**Pull request by SebTM**
Hey, I took care of the requested change in #113 but don't had permission to update the MR so why I created this 🙏🏻 Let me know if you would like something different - rebased on latest master.

### CODE REVIEW

File: README.org [hunk 1]
```diff
@@ -344,6 +344,44 @@
...
+            nixpkgs,
+            impermanence,
+            ...
+          }:
+          {
+            nixosConfigurations.sythe = nixpkgs.lib.nixosSystem {
+              system = "x86_64-linux";
+              modules = [
+                {
+                  imports = [ home-manager.nixosModules.home-manager ];
```
Comment (SebTM) [inline README.org:368]: Is this correct, regarding https://github.com/nix-community/impermanence/pull/181/files#diff-ecbc1aa90e9ff97a00b0b2aab1551bceee0c4d21993146bdcb1af4de31c9cac6R335?

**Comment by SebTM** [comment]
@talyz Any chance we can get this merged? 🙏🏻

**Review by Misterio77 — APPROVED** [review]
LGTM

**Comment by talyz** [comment]
Looks good to me! Can you please rebase on the latest master? I think merging now would create duplicate entries.

**Comment by SebTM** [comment]
> Looks good to me! Can you please rebase on the latest master?
Done 🙏🏻

**Comment by SebTM** [comment]
@talyz Can we get this merged? ✌🏻

**Pull request merged**